src/jws.rs

```diff
@@ -46,7 +54,7 @@ impl JwsBuilder {
         self
     }
 
-    /// Set the kid
+    #[cfg(test)]
```

Comment:
This was here for a reason. MS requires a very specific header when sending a Windows Hello assertion.

Reply:
Yep, that was my mistake :)